Issue with latest release (invalid memory address or nil pointer dereference) when running apko build

**Issue:**
./apko build examples/alpine-base.yaml test output2.tar
2022/03/21 01:00:11 loading config file: examples/alpine-base.yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x4e813e]

goroutine 1 [running]:
log.(*Logger).Printf(0x7ffe5b3bae73?, {0xb11abe?, 0xc0000e3d48?}, {0xc0000e3a88?, 0x2c?, 0xffaaaabfeaaaa?})
	log/log.go:202 +0x1e
chainguard.dev/apko/pkg/cli.BuildCmd({0x1?, 0x0?}, {0x7ffe5b3bae73, 0x4}, {0x7ffe5b3bae78, 0xb}, {0xc0000e3d48, 0xa, 0xa})
	chainguard.dev/apko/pkg/cli/build.go:113 +0x34c
chainguard.dev/apko/pkg/cli.Build.func1(0xc0001a2000?, {0xc000173da0, 0x3, 0x3?})
	chainguard.dev/apko/pkg/cli/build.go:62 +0x5a5
github.com/spf13/cobra.(*Command).execute(0xc0001a2000, {0xc000173d70, 0x3, 0x3})
	github.com/spf13/cobra@v1.4.0/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000193b80)
	github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.4.0/command.go:902
main.main()
	chainguard.dev/apko/main.go:24 +0x1e
	
	
**Go Version**: go version go1.18 linux/amd64. (also test with 1.17.8)
**APKO Version:**  Your branch is up to date with 'origin/main'.   0.2.1 (https://github.com/chainguard-dev/apko/commit/5201bc0fdef362743492765462064524bf52eb23) 
**Alpine version:** 
VERSION_ID=3.16.0_alpha20220316
PRETTY_NAME="Alpine Linux edge"

--- a/pkg/build/build.go
+++ b/pkg/build/build.go
@@ -133,6 +133,7 @@ func (bc *Context) runAssertions() error {
 func New(workDir string, opts ...Option) (*Context, error) {
 	bc := Context{
 		WorkDir: workDir,
+		Log:     log.New(log.Writer(), "apko (early): ", log.LstdFlags|log.Lmsgprefix),
 	}
 
 	for _, opt := range opts {
@@ -192,7 +193,7 @@ func (bc *Context) Refresh() error {
 }
 
 func (bc *Context) UpdatePrefix() {
-	bc.Log = log.New(log.Writer(), fmt.Sprintf("%s: ", bc.Arch.ToAPK()), log.LstdFlags|log.Lmsgprefix)
+	bc.Log = log.New(log.Writer(), fmt.Sprintf("apko (%s): ", bc.Arch.ToAPK()), log.LstdFlags|log.Lmsgprefix)
 }
 
 // Option is an option for the build context.

--- a/pkg/cli/build-minirootfs.go
+++ b/pkg/cli/build-minirootfs.go
@@ -71,6 +71,10 @@ func BuildMinirootFSCmd(ctx context.Context, opts ...build.Option) error {
 		return err
 	}
 
+	if err := bc.Refresh(); err != nil {
+		return err
+	}
+
 	if len(bc.ImageConfiguration.Archs) != 0 {
 		log.Printf("WARNING: ignoring archs in config, only building for current arch (%s)", bc.Arch)
 	}

--- a/pkg/cli/build.go
+++ b/pkg/cli/build.go
@@ -98,6 +98,10 @@ func BuildCmd(ctx context.Context, imageRef, outputTarGZ string, opts ...build.O
 		return err
 	}
 
+	if err := bc.Refresh(); err != nil {
+		return err
+	}
+
 	if bc.SBOMPath == "" {
 		dir, err := filepath.Abs(outputTarGZ)
 		if err != nil {

